status/integration_status.py

```diff
@@ -1,4 +1,185 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 """
 Definition of integration status table and CLI util to udpate status tables and 
 templates from yaml configs
 """
+
+import os
+from pathlib import Path
+
+from pydantic import Field
+
+from sparsezoo.utils.standardization import (
+    FeatureStatus,
+    FeatureStatusPage,
+    FeatureStatusTable,
+    write_status_pages,
+)
+
+
+class BaseTrainingStatusTable(FeatureStatusTable):
+
+    cli: FeatureStatus = Field()
+    api: FeatureStatus = Field()
+    dense_training: FeatureStatus = Field()
+    gradient_accumulation: FeatureStatus = Field()
+    DP: FeatureStatus = Field()
```

Comment:
nit: may be better to rename to lowercase (same for ddp, ema, amp). Having an all caps class field would be somewhat unexpected

Reply:
Thoughts on also adding formatting that would transform e.g. dense_training -> Dense Training